extensions/wikia/AdEngine/js/provider/gpt/googleTag.js

```diff
@@ -1,66 +1,66 @@
 /*global define*/
 /*jshint maxlen:125, camelcase:false, maxdepth:7*/
 define('ext.wikia.adEngine.provider.gpt.googleTag', [
+	'ext.wikia.adEngine.provider.gpt.googleSlots',
```

Comment:
Do you mind ordering it in alphabetical order? It's an old not written rule of ours :japanese_ogre: